portugues/colabora/comdiferencas/corrigedif/corrigedif.md

```diff
@@ -0,0 +1,2 @@
+Salvou o dia de muitos utilizadores corrigindo um bug! No entanto há que submeter essa alteraçao.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.